src/actions.rs

```diff
@@ -404,6 +404,100 @@ impl From<TouchActions> for ActionSequence {
     }
 }
 
+/// A sequence containing [`Wheel` actions](WheelAction) for a wheel device.
+#[derive(Debug, Clone)]
+pub struct WheelActions {
+    /// A unique identifier to distinguish this input source from others.
+    ///
+    /// Choose a meaningful string as it may be useful for debugging.
+    id: String,
+    /// The list of actions for this sequence.
+    actions: Vec<WheelAction>,
+}
+
+impl WheelActions {
+    /// Create a new `WheelActions` sequence.
+    ///
+    /// The id can be any string but must uniquely identify this input source.
+    pub fn new(id: String) -> Self {
+        Self {
+            id,
+            actions: Vec::new(),
+        }
+    }
+
+    /// Pushes a new action.
+    pub fn push(&mut self, action: WheelAction) {
+        self.actions.push(action);
+    }
+}
+
+impl From<WheelActions> for ActionSequence {
+    fn from(wa: WheelActions) -> Self {
+        ActionSequence(WDActions::ActionSequence {
+            id: wa.id,
+            actions: WDActions::ActionsType::Wheel {
+                actions: wa.actions.into_iter().map(|x| x.into_item()).collect(),
+            },
+        })
+    }
+}
+
+/// An action performed with a wheel device.
+///
+/// See [15.4.4 Wheel Actions](https://www.w3.org/TR/webdriver/#wheel-actions) of the
+/// WebDriver standard.
+#[derive(Debug, Clone)]
+pub enum WheelAction {
```

Comment:
Let's add the [`#[non_exhaustive]`](https://doc.rust-lang.org/reference/attributes/type_system.html#the-non_exhaustive-attribute) attribute to this so that we can add additional variants (like maybe `ScrollFromAbsolute` to have a different `origin`) in the future without breaking backwards compatibility.